web-marketplace/src/pages/ProjectFinished/ProjectFinished.tsx

```diff
@@ -61,11 +66,40 @@ const ProjectFinished: React.FC<React.PropsWithChildren<unknown>> = () => {
             }}
             linkComponent={Link}
           />
+          <CardItem
+            label="url"
+            value={{
+              name: projectUrl,
+              url: projectUrl,
+              children: (
+                <Box>
+                  <Link
+                    href={`${window.location.origin}/project-pages/${currentProjectId}/edit/settings`}
```

Comment:
not sure if we need the window.location.origin here
from the project create flow "finished" step, this opens up the settings page in a new tab, is this what we want @erikalogie?

Reply:
I don't think so indeed.